src/openforms/submissions/report.py

```diff
@@ -15,6 +16,35 @@
 logger = logging.getLogger(__name__)
 
 
+@dataclass
+class CoSigner:
+    co_sign_data: dict
+
+    @property
+    def representation(self) -> str:
+        return self.co_sign_data.get("representation", "")
```

Comment:
in general the patern `self.co_sign_data.get("...") or ""` is better to guarantee the type is correct if the key is absent or empty-ish (i.e. present but the value is `None`)

maybe a better approach would be a complete dataclass and have the `Report.co_signer` perform this normalization